app/services/CalculationService.scala

```diff
@@ -274,10 +274,11 @@ trait CalculationService {
 
   }
 
-  def determinePRRUsed (gain: Double, prrUsed: Option[Double]): Double = {
+  def determinePRRUsed (gain: Double, prrUsed: Option[Double], prr: String): Double = {
     prrUsed match {
-      case Some(a) if a < gain => round("up", a)
-      case Some(b) => gain
+      case a if prr.equals("Full") => gain
```

Comment:
Should this not initially match on None instead of case a? If they have selected Full they won't have put anything in prrValue? Unless prrValue is being set to 0 somewhere?